### PR TITLE
Added Stack Overflow tag

### PR DESCRIPTION
**Pull request by Dhiraj240**
Added Stack overflow tag in lmgtfy as per issue #345 
# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

**Comment by Dhiraj240** [comment]
Can anybody help in knowing why this Travis build has failed as i just changed the markdown file.Also here i am getting a update branch option should i click that ? Although i had rebased my branch with upstream.

**Comment by andrewda** [comment]
Hi @Dhiraj240, thanks for the PR! Check out [this link](http://coala.io/commit) to learn more about proper commit format (improper commit format will cause the builds to fail). Also, make sure to squash your commits into one!

**Comment by Dhiraj240** [comment]
@andrewda Okay, i will do it accordingly.

**Comment by jayvdb** [comment]
Ping @Dhiraj240 .

**Comment by Dhiraj240** [comment]
@jayvdb sorry, i had just joined my college after insem break so i was caught up with my college sessions.I will do the proper commit by morning.

**Comment by Dhiraj240** [comment]
@jayvdb Sorry for delay yesterday came back after attending hackathon,i hope i did the proper commit but still travis yelled at me.

**Comment by andrewda** [comment]
@Dhiraj240 if you scroll to the bottom of the Travis message, it provides some helpful hints on how to fix any problems. In this case:

```
**** GitCommitBear [Section: commit | Severity: NORMAL] ****
!    ! Shortlog of HEAD commit isn't in imperative mood! Bad words are 'Added'
```

You should be using the imperative tense in your commit message (documented [here](https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html#shortlog)), so say `Add StackOverflow tag` instead of `Added StackOverflow tag`. This is the typical format for a Git commit message.

**Comment by Dhiraj240** [comment]
@andrewda thank you , its done.

**Comment by Dhiraj240** [comment]
@li-boxuan I have done some changes as per your review and look what is this as i had tweaked just a markdown.

File: plugins/templates/lmgtfy.jinja2.md [hunk 1]
```diff
@@ -1 +1 @@
-Hey! You seem to have asked a question which could have been solved by other means as well. Click [this](https://www.lmgtfy.com/?q={{ query }}). When you ask a question, be sure to try out a few things first - you're in a much better position to help yourself than we are. Try googling, thinking, searching on GitHub or [git grep](https://git-scm.com/docs/git-grep) if you are looking for source code. [StackOverflow](http://stackoverflow.com/help/how-to-ask) has a good guide on how to ask questions in a way that allows us to help you even faster!
+Hey! You seem to have asked a question which could have been solved by other means as well. Click [this](https://www.lmgtfy.com/?q={{ query }}). When you ask a question, be sure to try out a few things first - you're in a much better position to help yourself than we are. Try googling, thinking, searching on GitHub or [git grep](https://git-scm.com/docs/git-grep) if you are looking for source code.Coala now has a fancy [coala](https://stackoverflow.com/questions/tagged/coala) StackOverflow tag, you can use it in [StackOverflow](http://stackoverflow.com/help/how-to-ask) to ask questions in a way that allows us to help you even faster!
```
Comment (li-boxuan) [inline plugins/templates/lmgtfy.jinja2.md]: Add a space after `... if you are looking for source code.` Also, coala is always written with a small 'c', even if it is the first word of a sentence.

**Comment by Dhiraj240** [comment]
Can anybody help me what all this is happening?

**Comment by jayvdb** [comment]
ack 332e8f9

**Comment by Dhiraj240** [comment]
is there is a case that until CI gets passed this PR will not be merged?

**Comment by jayvdb** [comment]
Possibly we wait until CI passes.  This patch isnt urgent.

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**